Transformations/utilities/reformat_graph.py

```diff
@@ -22,17 +21,9 @@ class RdfConverter():
     def __init__(self):
         self.graph = rdflib.Graph()
 
-    def convert(self, input_file, input_format, output_file='', output_format='xml'):
```

Comment:
Missing docstrings in this file.

Reply:
added docstrings [here](https://github.com/VCityTeam/UD-Graph/blob/aa846ff115fd7ce7eb5a3348a11464d4a3b35351/Transformations/utilities/reformat_graph.py#L26)